java/ql/automodel/test/AutomodelApplicationModeExtraction/Test.java

```diff
@@ -67,4 +68,15 @@ class OverrideTest extends Exception {
 	public void printStackTrace(PrintWriter writer) { // writer is a source candidate because it overrides an existing method
 		return;
 	}
+
+}
+
+class TaskUtils {
+	public FutureTask getTask() {
+		FutureTask ft = new FutureTask(() -> {
```

Comment:
I don't see this call to the `FutureTask` constructor anywhere in the test output. I'm guessing again this is because you created the before state by removing stuff from the after state?

Reply:
yes - I tried to avoid this kind of mistake but it must've slipped through 👍 kudos for spotting this :)